### ISSUE TITLE
[BUG] Protocol Error

### ISSUE THREAD

**Issue by shirshak55** [microsoft/playwright#10681]
**Context:**
- Playwright Version: ^1.17.1
- Operating System: Linux
- Node.js version: 17
- Browser: Firefox
- Extra: [any specific details about your environment]

Error when filling the form. frame.fill(selector, "")

```
Err elementHandle.fill: Protocol error (Page.dispatchKeyEvent): error in channel "content::page": exception while running method "dispatchKeyEvent" in namespace "page": Component returned failure code: 0xc1f30001 (NS_ERROR_NOT_INITIALIZED) [nsITextInputProcessor.keydown] _dispatchKeyEvent@chrome://juggler/content/content/PageAgent.js:564:13
_onMessage@chrome://juggler/content/SimpleChannel.js:165:37
receiveMessage@chrome://juggler/content/SimpleChannel.js:16:42
 _onMessage@chrome://juggler/content/SimpleChannel.js:149:16
receiveMessage@chrome://juggler/content/SimpleChannel.js:16:42
```

The err is spurious.

**Comment by pavelfeldman** [comment]
Is there a repro you could share?

**Comment by shirshak55** [comment]
@pavelfeldman thanks for the concern. There is no easy way to reproduce as it is too spurious. And the test runs for 1-2 days so IDK how to do it and it's an internal app. Also, This generally happens when we manually try to change something in headful mode.

**Comment by dgozman** [comment]
Seems to be the same issue as #5460.

**Comment by usersina** [comment]
I've managed to reproduce this actually with a rate of 100% albeit with a headful browser.

What I did is actually opening a page with a textarea in a headful Firefox instance with a debugger. I paused the execution of the code before filling the textarea and pressed some keyboard buttons (`F10` twice for example to open and close the menu bar in firefox). When I resumed the debugger, playwright throws this error when trying to type the message. I guess I "stole" the `dispatchKeyEvent` from playwright or something.
